backend/app/utils/configs/queries.py

```
@@ -8,8 +8,7 @@
 
 import json
 import datetime
-
-from app.utils.settings.config import settings
+from fastapi import Request
```

Ruff (0.8.0)

`fastapi.Request` imported but unused

Remove unused import: `fastapi.Request`

(F401)